uktrade/data-hub-frontend
# Reduce size of collection list filter controls

## Description of change

Reduces the size of collection list filters - text is now 16px

Also refactors collection list filters so they can be imported in a more concise way.

## Test instructions

Check out all the collection list pages and check the font size stays at 16px in all resolutions (mobile / tablet / desktop) as specced in https://uktrade.atlassian.net/browse/RR-169 

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/1234577/153410879-f93b0336-258c-4cec-b599-3a5bb422cdf3.png)

### After

![image](https://user-images.githubusercontent.com/1234577/153410774-0310732f-7a10-4f83-863d-d0d4f6ef2a47.png)

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
